core/templates/domain/topics_and_skills_dashboard/topics-and-skills-dashboard-backend-api.service.ts

```diff
@@ -17,27 +17,34 @@
   from the backend and to merge skills from the dashboard.
  */
 
-require(
-  'domain/topics_and_skills_dashboard/' +
-  'topics-and-skills-dashboard-domain.constants.ajs.ts');
+import { downgradeInjectable } from '@angular/upgrade/static';
+import { Injectable } from '@angular/core';
+import { HttpClient } from '@angular/common/http';
+import { TopicsAndSkillsDashboardDomainConstants } from
+  // eslint-disable-next-line max-len
+  '../topics_and_skills_dashboard/topics-and-skills-dashboard-domain.constants';
```

Comment:
Avoid using the linter pragma. Try to split the filepath here.

Reply:
I think it's not possible to split when importing.

Reply:
Yep, you're right. Resolving this comment, no action needed.